src/app/pages/ArticlePage/index.stories.tsx

```diff
@@ -229,6 +230,20 @@ export const ArticlePageWithPodcastNews = () => (
   />
 );
 
+export const ArticlePageWithTranscriptStage2 = () => (
```

Comment:
Can we indicate in the name what stage 2 means? (In this case it displays a transcript)